app/src/main/java/com/neongarage/Fragments/CountInFragment.java

```diff
@@ -157,43 +153,34 @@ public void handleSubmitClick(View v) {
         double tenTotal = tens.getTotal();
         double twentyTotal = twenties.getTotal();
         double dollarTot = oneTotal + fiveTotal + tenTotal + twentyTotal;
-        dollar.append("Ones: ");
-        dollar.append(df.format(oneTotal));
-        dollar.append("\nFives: ");
-        dollar.append(df.format(fiveTotal));
-        dollar.append("\nTens: ");
-        dollar.append(df.format(tenTotal));
-        dollar.append("\nTwenties: ");
-        dollar.append(df.format(twentyTotal));
+        dollar.append("Ones: ").append(df.format(oneTotal));
+        dollar.append("\nFives: ").append(df.format(fiveTotal)).append("\nTens: ").append(df.format(tenTotal)).append("\nTwenties: ").append(df.format(twentyTotal));
         dollarTotal.setText(dollar);
         dollar.setLength(0);
 
         //Grand total
         double grandTot = dollarTot + coinValue;
-        grand.append("Grand Total: ");
-        grand.append(df.format(grandTot));
+        grand.append("Grand Total: ").append(df.format(grandTot));
         grandTotal.setText(grand);
         grand.setLength(0);
 
         //Difference
         if (grandTot > 150.45) {
             double difference = grandTot - 150.45;
             //Shay says this is green....
-            resultText.setTextColor(Color.rgb(0, 128, 0));
-            result.append("You are over ");
-            result.append(df.format(Math.ceil(difference)));
+            resultText.setTextColor(getResources().getColor(R.color.overColor));
+            result.append("You are over ").append(df.format(Math.ceil(difference)));
             resultText.setText(result);
             result.setLength(0);
         } else if (grandTot < 149.95) {
             double difference = 149.95 - grandTot;
             //Apparently this is red
-            resultText.setTextColor(Color.rgb(128, 0, 0));
-            result.append("You are under ");
-            result.append(df.format(Math.ceil(difference)));
+            resultText.setTextColor(getResources().getColor(R.color.underColor));
+            result.append("You are under ").append(df.format(Math.ceil(difference)));
             resultText.setText(result);
             result.setLength(0);
         } else {
-            resultText.setTextColor(Color.rgb(0, 0, 128));
+            resultText.setTextColor(getResources().getColor(R.color.perfectColor));
```

Comment:
Ideally you would want to use the Attributes defined colors... This is a little complicated and you can do it in a follow up PR. But 
1) Define colors for each theme in your colors file (you only have one theme) and you already did this
<?xml version="1.0" encoding="utf-8"?>
<resources>}
    ...
    <color name="overColor">#800000</color>
    <color name="underColor">#008000</color>
    <color name="perfectColor">#000080</color>
    ...
</resources>
2) Create file res/values/attrs.xml with contents:
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <attr name="overColor" format="reference" />
    <attr name="underColor" format="reference" />  
    <attr name="perfectColor" format="reference" />
</resources>
3) Now if you styles.xml define:
    <!-- Base application theme. -->
    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
        <!-- Customize your theme here. -->
        ...
         <item name="overColor">@color/overColor</item>
        <item name="underColor">@color/underColor</item>
        <item name="perfectColor">@color/perfectColor</item>
    </style>

4) Use the color in a drawable:
<color android:color="?attr/my_color"/>

Reply:
The reason you're doing all this extra work is so that in the future you can create a new style for a dark theme and not change everything